src/main/java/com/rybka/config/PropertyInfo.java

```diff
@@ -1,8 +1,15 @@
 package com.rybka.config;
 
+import java.io.File;
+
 public final class PropertyInfo {
-    public static final String PROPERTY_FILE_NAME = "application.properties";
+    public static final String RESOURCE_PATH = "src" + File.separator + "main" + File.separator + "resources" + File.separator;
```

Comment:
Don't think it's a good idea to store in this path. 

Let user configure path from export folder in application.properties.

Reply:
fixed